app/controllers/tags_controller.rb

```diff
@@ -61,6 +61,10 @@ def suggest
 
   private
 
+  def tags
+    @tags ||= Tag.direct.includes(:sponsorship).limit 100
```

Comment:
Interesting that Rubocop didn't flag this. We basically reserve method calls without parens for DSL-like class "macros", like `belongs_to`, etc. Not a big deal, but seeing two different styles used on the same line sends me on an OCD spiral 😆 

```suggestion
    @tags ||= Tag.direct.includes(:sponsorship).limit(100)
```

Reply:
haha makes sense! No worries, It's admittedly a habit of mine. Whether it's good or bad is debatable 😆